src/Type/Definition/FieldArgument.php

```diff
@@ -87,7 +95,7 @@ public function getType() : Type
 
     public function defaultValueExists() : bool
     {
-        return array_key_exists('defaultValue', $this->config);
+        return $this->defaultValueExists;
```

Comment:
Looks like this reverts some changes made recently (see https://github.com/webonyx/graphql-php/pull/566#discussion_r339321880). Is it intentional?

Reply:
I did it to align the implementation with `InputObjectField`.

Let's make all the same: https://github.com/webonyx/graphql-php/pull/585